sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2_and_later/sample_analyze_layout.py

```diff
@@ -60,10 +60,10 @@ def analyze_layout():
         )
     result = poller.result()
 
-    for style in result.styles:
-        print(
-            f"Document contains {'handwritten' if style.is_handwritten else 'no handwritten'} content"
-        )
+    if any([style.is_handwritten for style in result.styles]):
```

Comment:
Can you update the async sample as well?

Reply:
Sure.